Undo the disable of source maps for non-chrome browsers

vegegoku opened the pull request:
Nowadays, all modern browsers support sourcemaps by default, so there is no need to have exception configuration anymore.

vegegoku committed:
Undo the disable of source maps for non-chrome browsers

Nowadays, all modern browsers support sourcemaps by default, so there is no need to have exception configuration anymore.

niloc132 reviewed (commented):
Worth testing if this is what was breaking https://github.com/gwtproject/gwt/issues/9759 - since this claims that only chrome/safari support sourcemaps (clearly untrue, hasn't been true for a very long time), this might prevent the symbolmaplinker from correctly producing output when collapsed?

vegegoku committed:
Undo the disable of source maps for non-chrome browsers

Nowadays, all modern browsers support sourcemaps by default, so there is no need to have exception configuration anymore.

Fixes: #9759

niloc132 commented on user/src/com/google/gwt/core/CoreWithUserAgent.gwt.xml:
```diff
@@ -12,22 +12,13 @@
 <!-- limitations under the License.                                         -->
 
-<!-- Deferred binding rules for core classes based on user agent. -->
+<!-- Deferred binding rules for core classes. -->
```
If this is no longer based on user agent, can we move the remaining replace-with to StackTrace.gwt.xml where it probably belongs?

That would leave this file empty, and we can let users remove it from their own projects, eventually removing it here.

niloc132 commented:
Update description to list any issues fixed by this, and summarize changes, then we can merge?
